src/App.test.tsx

```diff
@@ -7,13 +7,13 @@ import Routes from './routes';
 it('has a header', () => {
   const wrapper = shallow(<App />);
 
-  expect(wrapper.containsMatchingElement(<header />)).toBe(true);
+  expect(wrapper.containsMatchingElement(<header>Header</header>)).toBe(true);
```

Comment:
Actually, we check footer elements existence, so no need to pass all element details here.